src/test/integration/integration-test.py

```diff
@@ -25,50 +25,64 @@ def test_get_authority_onids(self, endpoint='/spendingauthority'):
         for onid in self.test_cases['valid_authority_onids']:
             params = {'onid': onid}
             response = utils.make_request(self, endpoint, 200, params=params)
-            self.checking_response_schema(response, onid)
-            self.checking_spending_limits(response)
-
-    # Test case: GET /spendingauthority with valid authority onids
-    def checking_response_schema(self, response, onid):
-        spending_schema = utils.get_resource_schema(
-            self, 'SpendingAuthorityResource'
-        )
-        utils.check_schema(self, response, spending_schema)
-        response_data = response.json()['data']
-        actual_onid = response_data['id']
-        logging.debug(f'Request to made to {onid.lower()}')
-        logging.debug(f'Response for {actual_onid.lower()}')
-        self.assertEqual(actual_onid.lower(), onid.lower())
+            response_data = response.json()['data']
+            # Test case: GET /spendingauthority with valid authority onids
+            spending_schema = utils.get_resource_schema(
+                self, 'SpendingAuthorityResource'
+            )
+            utils.check_schema(self, response, spending_schema)
+            response_data = response.json()['data']
+            actual_onid = response_data['id']
+            logging.debug(f'Request to made to {onid}')
+            logging.debug(f'Response for {actual_onid}')
+            self.assertEqual(actual_onid.lower(), onid.lower())
+
+            self.checking_spending_limits(response_data['attributes'])
 
     # Test case: GET /spendingauthority spending limits
-    def checking_spending_limits(self, response):
+    def checking_spending_limits(self, attributes):
         # Checking that the lists of spending limits and indexes are unique
         # Checking valid onids with spending authority only
-        # To make sure that attributes is not empty
-        response_data = response.json()['data']
-        attributes = response_data['attributes']
-        self.assertTrue(attributes)
-        limits = attributes['limits']
-        # Checking that limits has at least one index
-        logging.debug(f'limits: {len(limits)}')
-        self.assertTrue(limits)
-        spendinglimit = []
+        # To make sure that attributes and index are not empty
+        non_empty_list_dict = {
+            'attributes': attributes,
+            'limits': attributes['limits']
+        }
+
+        for key, non_empty_list in non_empty_list_dict.items():
+            logging.debug(f'{key}: {len(non_empty_list)}')
```

Comment:
Again, the debug message is ambiguous here and it'll be challenging to understand what does this message mean if we run into an issue. Can you make it more descriptive as well?